CONTRIBUTING.md

```diff
@@ -40,15 +40,15 @@ There are two types of automated tests:
 * unit tests, implemented using [PHPUnit](http://phpunit.de/)
 * functional tests, implemented using [Behat](http://behat.org)
```

Comment:
This is a confusing one for all of us. But grammatically if If the text that follows the bullet point is not a proper sentence, it doesn’t need to begin with a capital letter, nor end with a period.